docs/registry/rhoas-cli-getting-started-registry/README.adoc

```diff
@@ -84,6 +85,32 @@ You can get started with {product-long-registry} by doing the following tasks:
 [role="_abstract"]
 * You've installed the `rhoas` CLI. For more information, see {base-url}{rhoas-cli-installation-url}[_Installing and configuring the rhoas CLI_^].
 
+[id="proc-creating-service-context_{context}"]
+== Creating a Service Context
```

Comment:
It might be read as user needing to create context to make any operations where default context is already present. I think we need separate guide for the context on it's own. For service registry and kafka guides we should probably showcase:

- status command 
- generate-config command

Reply:
The registry and kafka guides will still have `context` references. Will that be okay?